### DISCUSSION TITLE
small font for CFB

### DISCUSSION THREAD

**Discussion by gkujawsk** [zephyrproject-rtos/zephyr#46482] [Q&A] [answered]
Hello guys, 
CFB by default has 16,24 and 32 pixels fonts only. Can anybody recommend the ttf font, that's gonna look good after being translated into format supported by CFB? I've tried a couple of these using /dev/urandom strategy backed by my "ML" ;) but still can't anything that displays nicely. 

-- 
/G

**Comment by Oleh-Kravchenko** [comment, marked as answer]
Hello @gkujawsk,
I use CGAthin from https://int10h.org/oldschool-pc-fonts/fontlist/
```
GENERATE_CFB_FONT_FOR_TARGET(app
"${CMAKE_CURRENT_SOURCE_DIR}/fonts/MxPlus_IBM_CGAthin.ttf"
"${ZEPHYR_BINARY_DIR}/include/generated/cfb_font_dice.h"
8 8 -s 8)
```

**Comment by gkujawsk** [comment]
Thank you. I will give it a shot.
